### PR TITLE
fix tidb/cockroach compatibility issue

### PR DESCRIPTION
**Pull request by liugangnhm**
* fix tidb/cockroach compatibility issue

### CODE REVIEW

**Comment by liugangnhm** [comment]
PTAL @ngaut @nieyy

File: README.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
...
 Experiential storage engine for TiDB 
 
-* Add cockroach support for tidb(see [cockroach](./store/cockroach/README.md))
\ No newline at end of file
+* Add cockroach support for tidb(see [cockroach](./store/cockroach/README.md))
+
+** Note **
+
+I don't know how to use secure mode(no usage document yet), so I disable secure mode.
+Please start cockroach with `insecure=true` parameter.
```
Comment (nieyy) [inline README.md]: I prefer use secure mode for CockroachDB, this is just a temporary solution, my opinion is add PR later after revised this issue.

**Comment by nieyy** [comment]
Maybe we can use `glock` to resolve the dependency issue.

**Comment by ngaut** [comment]
LGTM

**Comment by nieyy** [comment]
LGTM

**Pull request merged**